services/lsst-login.rst

```diff
@@ -89,9 +89,93 @@ When using an ``lsst-login`` node as a "jump host" you may also wish to configur
       ProxyJump lsst-login01.ncsa.illinois.edu
       DynamicForward yourportnumber
 
-You may also wish to reuse a single connection to/through an ``lsst-login`` node via a control socket/multiplexing. See for example
+You may also wish to reuse a single connection to/through an ``lsst-login`` node via a control socket/multiplexing. This means you can authenticate to the login node once, and reuse that connection in a 2nd terminal without authenticating again. See for example
```

Comment:
While this is accurate, the word **multiplexing** made me think of **terminal multiplexing** (`screen`, `tmux`, etc). I'd consider dropping the word **multiplexing**.

Reply:
I think since it's the correct term and what is used in the official docs, I should remain.